docs/designers-developers/developers/tutorials/readme.md

```diff
@@ -1,13 +0,0 @@
-# Tutorials
```

Comment:
Moved to `docs/tutorials/README.md`. It looks like GitHub didn't pick up this rename.